### ISSUE TITLE
Mattermost MacOS Desktop 

### ISSUE THREAD

**Issue by metehankaraca** [mattermost/mattermost-server#20321]
Per Mattermost guidelines, GitHub issues are for bug reports: <https://handbook.mattermost.com/contributors/contributors/ways-to-contribute/>.

For troubleshooting see: https://forum.mattermost.com/.
For feature proposals see: https://www.mattermost.com/feature-ideas/

If you've found a bug--something appears unintentional--please follow these steps:

1. Confirm you’re filing a new issue. [Search existing tickets in Jira](https://mattermost.atlassian.net/jira/software/c/projects/MM/issues/) to ensure that the ticket does not already exist.
2. Confirm your issue does not involve security. Otherwise, please see our [Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
3. [File a new issue](https://github.com/mattermost/mattermost-server/issues/new) using the format below. Mattermost will confirm steps to reproduce and file in Jira, or ask for more details if there is trouble reproducing it. If there's already an existing bug in Jira, it will be linked back to the GitHub issue so you can track when it gets fixed. 

#### Summary
Bug report in one short sentence
Sending a message

#### Steps to reproduce
How can we reproduce the issue (what version are you using?)
Version 5.0.4 (5.0.4.17220)

#### Expected behavior
Describe your issue in detail
Send message button in Turkish language macos desktop version is wrong

#### Observed behavior (that appears unintentional) 
What did you see happen? Please include relevant error messages and/or screenshots.

#### Possible fixes
If you can, link to the line of code that might be responsible for the problem
<img width="253" alt="Screen Shot 2022-05-30 at 23 11 26" src="https://user-images.githubusercontent.com/55768892/171054582-c652353f-0e8e-4402-8c4b-37f6825038f3.png">
![IMG_4087](https://user-images.githubusercontent.com/55768892/171054586-52974ede-ac8d-4367-bbf8-0f110012c1fc.jpg)

**Comment by amyblais** [comment]
@metehankaraca Translations are done by volunteers via the translations server https://translate.mattermost.com/languages/. Would you be open to signing up and suggesting a translation edit via the server?

**Comment by amyblais** [comment]
@metehankaraca Please see discussion at https://community.mattermost.com/core/pl/6g6xbandtjr1jejk7ur46s3j5w. If you suggest an edit, our translations team can help approve it.

**Comment by metehankaraca** [comment]
https://translate.mattermost.com/languages/  . 
I looked in the translation section but couldn't find the wrong translation. How can I access the wrong section in the app?

**Comment by amyblais** [comment]
@metehankaraca Do you see the same issue on Desktop App v5.1?

**Comment by metehankaraca** [comment]
@amyblais yes still available. button to send a private message to members in the server.

**Comment by ctlaltdieliet** [comment]
Can be closed.
String has changed in [Weblate](https://translate.mattermost.com/translate/mattermost/mattermost-webapp_master/tr/?checksum=74986f9f214acb4)

**Comment by metehankaraca** [comment]
Thanks
